app/uk/gov/hmrc/childcarecalculatorfrontend/utils/SessionDataOverwriter.scala

```diff
@@ -1,45 +0,0 @@
-/*
- * Copyright 2017 HM Revenue & Customs
```

Comment:
@versatil56 Seems this is not being used anywhere in the code. Could you please explain what this class is used for ? If its not used at all then we can delete it and the spec as well.

Reply:
Got the reply from Juan and it can be deleted as its not being used in the code.